src/MediaManagerBundle/Resources/public/js/mediaManager.js

```diff
@@ -67,14 +67,21 @@
             var self = this;
             $(this.treeContainer).jstree({
                 'plugins': [
-                    'cookies', 'dnd', 'hotkeys', 'contextmenu', 'state', 'wholerow', 'types'
+                    'dnd', 'contextmenu', 'state', 'wholerow', 'types'
```

Comment:
Cookies and hotkeys modules where removed here. Is this o.k.?

Reply:
yes, these plugins no longer exist.